src/pages/Address/Cells.tsx

```diff
@@ -185,21 +195,31 @@ const Cell: FC<{ cell: LiveCell }> = ({ cell }) => {
     index: `0x${cell.cellIndex.toString(16)}`,
   }
 
+  const handleClick = () => {
+    history.push(link)
+  }
+
   return (
-    <li key={cell.txHash + cell.cellIndex} className={styles.card}>
+    <li
+      key={cell.txHash + cell.cellIndex}
+      className={classNames(styles.card, styles.pointer)}
+      onClick={handleClick}
+      onKeyDown={handleClick}
+    >
       <h5>
-        <a href={link}>{title}</a>
-
+        {title}
         <button type="button" className={styles.copy} data-detail={JSON.stringify(outPoint)} onClick={handleCopy}>
           <CopyIcon />
         </button>
-        <span title={`${ckb} CKB`}>{`${ckb} CKB`}</span>
+        <span className={styles.pointer} title={`${ckb} CKB`}>{`${ckb} CKB`}</span>
       </h5>
       <div className={styles.content}>
         {typeof icon === 'string' ? <img src={icon} alt={assetName ?? 'sudt'} width="40" height="40" /> : null}
         {icon && typeof icon !== 'string' ? icon : null}
         <div className={styles.fields}>
-          <div className={styles.assetName}>{assetName}</div>
+          <div className={styles.assetName} onClick={handleAssetNameClick} onKeyDown={undefined}>
```

Comment:
It can be done by using a `a` tag or a `Link` component

Reply:
The page will redirect to the transaction page when click the card, the page will redirect to the asset page like xUDT page when click the name of asset live cell card, and the content will be copied to the clipboard when the copy button is clicked. And I found that if I use the `a`, it will be abnormal when click the asset name

Reply:
`And I found that if I use the a, it will be abnormal when click the asset name` what problem will it be if `a` is used.

Reply:
Any update on this issue

Reply:
Any update on this conversation @Daryl-L

Reply:
> > And I found that if I use the a, it will be abnormal when click the asset name
> 
> what problem will it be if `a` is used?

By using `a` tag, when I click the link, the page will redirect to the page as clicking the whole card. So I just use to prevent the redirection.

Reply:
It's caused by `event bubble` and can be fixed by adding a event handler with `stopPropagation`

Ref: 
- stop propagation: https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation